lib/components/artist/shows/medium_image_show.js

```diff
@@ -11,37 +11,35 @@ import OpaqueImageView from '../../opaque_image_view'
 import ShowMetadata from './metadata'
 import SwitchBoard from '../../../modules/switch_board';
 
-const windowDimensions = Dimensions.get('window');
-const imageWidth = (windowDimensions.width - 120)/3;
-
 class MediumImageShow extends React.Component {
   handleTap() {
     SwitchBoard.presentNavigationViewController(this, this.props.show.href);
   }
 
   render() {
+    const window = Dimensions.get('window');
+    const imageWidth = (window.width - 120) / 3;
+    const imageHeight = Math.floor(imageWidth / 1.5);
+
     return (
       <TouchableWithoutFeedback onPress={this.handleTap.bind(this)}>
         <View style={{margin: 10, width : imageWidth}}>
-          <OpaqueImageView imageURL={this.props.show.meta_image.cropped.url} style={{width: this.props.relay.variables.imageWidth, height: this.props.relay.variables.imageHeight, marginBottom: 5}} />
-          <ShowMetadata style= {{ width: this.props.relay.variables.imageWidth }} show={this.props.show} />
+          <OpaqueImageView imageURL={this.props.show.meta_image.url}
+                              style={{ width: imageWidth, height: imageHeight, marginBottom: 5 }} />
+          <ShowMetadata style= {{ width: imageWidth }} show={this.props.show} />
         </View>
       </TouchableWithoutFeedback>
     );
   }
 }
 
 export default Relay.createContainer(MediumImageShow, {
-  initialVariables: { imageWidth: Math.floor(imageWidth), imageHeight: Math.floor(imageWidth / 1.5) },
-
   fragments: {
     show: () => Relay.QL`
       fragment on PartnerShow {
         href
         meta_image {
-          cropped(width: $imageWidth, height: $imageHeight) {
-            url
-          }
+          url(version: "large")
```

Comment:
so we're using the large image url everywhere, and just sizing the view accordingly for large, medium, and small images, right?